### PR TITLE
Html improvements

### PR DESCRIPTION
**Pull request by ShimShtein**
Fixes #8785 - add thead to html tables

### CODE REVIEW

File: app/helpers/layout_helper.rb [hunk 1]
```diff
@@ -292,7 +292,7 @@ def alert(opts = {})
       result
     end
   end
-
+
```
Comment (ohadlevy) [inline app/helpers/layout_helper.rb]: nitpick: extra whitespaces.

File: app/views/compute_profiles/show.html.erb [hunk 2]
```diff
@@ -9,17 +9,19 @@
...
-                        edit_compute_profile_compute_attribute_path(@compute_profile.to_param, compute_attribute.id)
-                      else
-                        new_compute_profile_compute_resource_compute_attribute_path(@compute_profile.to_param, compute_resource.to_param)
-                      end %>
+  <thead>
+    <tr>
+      <th><%= _("Compute Resource") %></th>
+      <th><%= _("VM Attributes (%s)") % @compute_profile.name %></th>
+    </tr>
+  <thead>
```
Comment (ohadlevy) [inline app/views/compute_profiles/show.html.erb]: should be `</thead>`

File: app/views/config_templates/index.html.erb [hunk 3]
```diff
@@ -8,7 +8,7 @@
 %>
 
 <table class="table table-bordered table-striped table-two-pane">
-  <tr>
+  <thead>
     <tr>
```
Comment (ohadlevy) [inline app/views/config_templates/index.html.erb:13]: nitpick: wrong indentation.

File: app/views/hosts/_overview.html.erb [hunk 4]
```diff
@@ -1,7 +1,9 @@
 <table id="properties_table" class="table table-bordered table-striped">
-  <tr>
-    <th colspan="2"><%= _('Properties') %></th>
-  </tr>
+  <thead>
```
Comment (ohadlevy) [inline app/views/hosts/_overview.html.erb:2]: nitpick: identation

File: app/views/puppetclasses/index.html.erb [hunk 5]
```diff
@@ -3,15 +3,17 @@
...
-    <th><%= sort :name, :as => s_("Puppetclass|Name") %></th>
-    <th><%= sort :environment, :as => _("Environments and documentation") %></th>
-    <th><%= _('Host groups') %></th>
-    <th><%= sort :total_hosts, :as => _('Hosts'), :default => "DESC" %></th>
-    <th><%= sort :params_count, :as => _('Parameters'), :default => "DESC" %></th>
-    <th><%= sort :variables_count, :as => _('Variables'), :default => "DESC" %></th>
-    <th></th>
-  </tr>
+  <thead>
+	  <tr>
```
Comment (ohadlevy) [inline app/views/puppetclasses/index.html.erb]: nitpick: identation

File: app/views/reports/_list.html.erb [hunk 6]
```diff
@@ -1,17 +1,19 @@
 <table class="table table-bordered table-striped ellipsis">
-  <tr>
-    <% unless params[:host_id] %>
+  <thead>
+    <tr>
+      <% unless params[:host_id] %>
       <th><%= sort :host, :as => _("Host") %></th>
```
Comment (ohadlevy) [inline app/views/reports/_list.html.erb:5]: nitpick: identation

File: app/views/smart_proxies/index.html.erb [hunk 7]
```diff
@@ -4,12 +4,14 @@
...
 
 <table class="table table-bordered table-striped table-two-pane">
-  <tr>
-    <th><%= s_("SmartProxy|Name") %></th>
-    <th><%= s_("SmartProxy|Url") %></th>
-    <th><%= _("Features") %></th>
-    <th></th>
-  </tr>
+  <thead>
+	  <tr>
```
Comment (ohadlevy) [inline app/views/smart_proxies/index.html.erb]: nitpick: identation

File: app/views/subnets/index.html.erb [hunk 8]
```diff
@@ -4,6 +4,7 @@
 <% title_actions display_link_if_authorized(_("New Subnet"), hash_for_new_subnet_path) %>
 
 <table class="table table-bordered table-striped table-two-pane">
+  <thead>
   <tr>
```
Comment (ohadlevy) [inline app/views/subnets/index.html.erb]: nitpick: identation

**Comment by ohadlevy** [comment]
if we add `<thead>` I wonder if we should also add `<tbody>` to align?

**Comment by ShimShtein** [comment]
OK, fixed the tbody also.
Plus some formatting

**Comment by ohadlevy** [comment]
[test]

**Comment by ohadlevy** [comment]
[test]

**Comment by ohadlevy** [comment]
merge as e77dd57 with minor changes, thanks @ShimShtein

**Pull request closed without merging**